android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -392,9 +390,12 @@ public void isMeetingConnected(Promise promise) {
       return;
     }
 
-    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
-
-    promise.resolve(inMeetingService.isMeetingConnected());
+    UiThreadUtil.runOnUiThread(new Runnable() {
+      @Override
+      public void run() {
+        promise.resolve(zoomSDK.getInMeetingService().isMeetingConnected());
+      }
+    });
```

Comment:
this and `isMeetingHost` actually does not need to be wrapped - I've tested it myself.
But I guess that's fine to have this pattern consistently across this file